### PR TITLE
RDoc-2082-Reworded and moved warning to top, added image and code sam…

### PR DESCRIPTION
**Pull request by serezhub**
…ples

### CODE REVIEW

File: Documentation/5.1/Raven.Documentation.Pages/indexes/indexing-related-documents.dotnet.markdown [hunk 1]
```diff
@@ -1,44 +1,95 @@
...
 
-To extend indexing capabilities and simplify many scenarios, we have introduced the possibility for indexing related documents.
+{NOTE: }
 
-## Example I
+* To extend indexing capabilities and simplify many scenarios, we have introduced the possibility to 
+  index related documents with `LoadDocument`.
+
+* [Include( )](../client-api/session/loading-entities#load-with-includes) 
+  is an alternative session CRUD method that can pull data from related documents while reducing expensive trips to the disk.
```
Comment (ayende) [inline Documentation/5.1/Raven.Documentation.Pages/indexes/indexing-related-documents.dotnet.markdown:10]:
```suggestion
  is an alternative session CRUD method that can pull data from related documents while reducing expensive trips to the server.  
```

File: Documentation/5.1/Raven.Documentation.Pages/indexes/indexing-related-documents.dotnet.markdown [hunk 2]
```diff
@@ -1,44 +1,95 @@
...
+
+* [Include( )](../client-api/session/loading-entities#load-with-includes) 
+  is an alternative session CRUD method that can pull data from related documents while reducing expensive trips to the disk.  
+
+* People who are accustomed to relational models but want the agility and efficiency of document-based models 
+  should understand that [documents are most effective when they generally stand on their own](https://ravendb.net/learn/inside-ravendb-book/reader/4.0/3-document-modeling) 
+  and [relations are harmless exceptions](https://ravendb.net/learn/inside-ravendb-book/reader/4.0/10-static-indexes-and-other-advanced-options#indexing-referenced-data) to the rule.
+
+{INFO: Important} 
+Indexes are updated automatically whenever related documents change.
```
Comment (ayende) [inline Documentation/5.1/Raven.Documentation.Pages/indexes/indexing-related-documents.dotnet.markdown:17]:
```suggestion
The indexes will be updated automatically whenever related documents change. 
```

File: Documentation/5.1/Raven.Documentation.Pages/indexes/indexing-related-documents.dotnet.markdown (hunk 1 above)
Comment (serezhub) [reply Documentation/5.1/Raven.Documentation.Pages/indexes/indexing-related-documents.dotnet.markdown]: done

File: Documentation/5.1/Raven.Documentation.Pages/indexes/indexing-related-documents.dotnet.markdown (hunk 2 above)
Comment (serezhub) [reply Documentation/5.1/Raven.Documentation.Pages/indexes/indexing-related-documents.dotnet.markdown]: done

**Review by ayende — APPROVED** [review]

**Pull request merged**